### PR TITLE
Add tests for utils/mediumAvatar

### PR DESCRIPTION
**Pull request by nashvail**

### CODE REVIEW

File: src/utils/__tests__/mediumAvatar-test.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import mediumAvatar from '../mediumAvatar';
```
Comment (borisyankov) [inline src/utils/__tests__/mediumAvatar-test.js]: You might want to rename to `getMediumAvatar`

File: src/utils/__tests__/mediumAvatar-test.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+describe('mediumAvatar', () => {
+  test('If avatarUrl does not contain IMAGE_NAME.png returns avatarUrl as is ', () => {
```
Comment (borisyankov) [inline src/utils/__tests__/mediumAvatar-test.js]: NIT: Use small caps for first letter.

File: src/utils/__tests__/mediumAvatar-test.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+    const url = '/avatar/IMAGE_NAME/';
+    expect(mediumAvatar(url)).toEqual(url);
+  });
```
Comment (borisyankov) [inline src/utils/__tests__/mediumAvatar-test.js]: Try to follow Arrange-Act-Assert (Google it)
It is kinda invisible, but try to have each step on different line, and if long separated by an empty line, see the other tests.

File: src/utils/__tests__/mediumAvatar-test.js [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+
+  test('If avatarUrl contains IMAGE_NAME.png replaces .png with -medium.png in the URL', () => {
```
Comment (borisyankov) [inline src/utils/__tests__/mediumAvatar-test.js]: I don't think the test descriptions are clear enough. These are crucial; it is worth taking the time to make them clear.

For example, I am not 100% sure what you mean by the two descriptions here, though I suspect the file extension is the key. Try to explain it more high-level and you don't need to include the exact names and behavior (we should be able to tell by the code itself).

**Comment by borisyankov** [comment]
You can also add Flow types to the mediumAvatar.js file, as it is small and a good place to start 👍

**Comment by nashvail** [comment]
@borisyankov many thanks for the comments. I read pieces on how to write better tests and tried improving these. Please have a look. Thanks again :)

**Comment by borisyankov** [comment]
Looks good now 👍

**Pull request merged**